packages/paginated-table/README.md

```diff
@@ -1,5 +1,8 @@
 # PAGINATED TABLE COMPONENT DESIGN
 
+```diff
+- After updating to latest version remove transform: 'rotateX(180deg)) in red
```

Comment:
@rana22 this one looks bit confusing. Can you add details from where to remove this syntax or what specific changes to make? 

https://github.com/CBIIT/bento-frontend/blob/table_version_2/packages/paginated-table/README.md

Reply:
This style is copied over from Bento 3.0, the style does not add any style to table. 
- rotateX(180deg))  in bento-core table component flips the table so to correct all the bento project is currently adding rotateX(180deg)) to flip back to correct position.